### PR TITLE
asr speech recognition example

### PR DESCRIPTION
**Pull request by husenzhang**
## Description
Provides an example of ASR (automated speech recognition) model serving.  
Input is a wav file; output text translation of that wav.

## Type of change

Please delete options that are not relevant.

- [X] New feature (non-breaking change which adds functionality)

## Checklist:

- [X] Did you have fun?
- [X] Have you added tests that prove your fix is effective or that this feature works?
- [X] Has code been commented, particularly in hard-to-understand areas?
- [X] Have you made corresponding changes to the documentation?

### CODE REVIEW

File: examples/asr_rnnt_emformer/README.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+### ASR (Automated Speech Recognition) Example
+
+In this example we use torchserve to serve a ASR model that convert wav to text.
```
Comment (maaquib) [inline examples/asr_rnnt_emformer/README.md:3]: Thanks for contributing @husenzhang Can you add a few more details to the README as to what the scripts are doing and some example input output scenatios

Comment (husenzhang) [reply examples/asr_rnnt_emformer/README.md]: Thanks @maaquib.  I have updated [README](https://github.com/husenzhang/torch_serve/blob/master/examples/asr_rnnt_emformer/README.md) to include more details.

File: examples/asr_rnnt_emformer/requirements.txt [hunk 2]
```diff
@@ -0,0 +1,3 @@
+--extra-index-url https://download.pytorch.org/whl/cpu
+torchaudio
```
Comment (lxning) [inline examples/asr_rnnt_emformer/requirements.txt:2]: by default, torchserve install it via ts_scripts/install_dependency.py

File: examples/asr_rnnt_emformer/handler.py [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+        """
+        Invoke by TorchServe for prediction request.
+        Do pre-processing of data, prediction using model and postprocessing of prediciton output
+        :param data: Input data for prediction
+        :param context: Initial context contains model server system properties.
+        :return: prediction output
+        """
+        if isinstance(data, list):
+            data = data[0]
+        data = data.get("data") or data.get("body")
```
Comment (lxning) [inline examples/asr_rnnt_emformer/handler.py:55]: This is not able to handle batching. ref [example](https://github.com/pytorch/serve/blob/master/ts/torch_handler/vision_handler.py#L38)

Comment (msaroufim) [reply examples/asr_rnnt_emformer/handler.py:55]: To be fair neither does our default handler , for an example I think this is fine

File: examples/asr_rnnt_emformer/requirements.txt (hunk 2 above)
Comment (msaroufim) [reply examples/asr_rnnt_emformer/requirements.txt:2]: It's fine it's a tutorial

**Review by msaroufim — APPROVED** [review]

File: examples/asr_rnnt_emformer/00_save_jit_model.sh [hunk 4]
```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+CONTAINER=pytorch/torchserve:0.6.0-cpu
```
Comment (chauhang) [inline examples/asr_rnnt_emformer/00_save_jit_model.sh:3]: Please update this to use the current latest version of TorchServe

File: examples/asr_rnnt_emformer/01_create_model_archive.sh [hunk 5]
```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+CONTAINER=pytorch/torchserve:0.6.0-cpu
```
Comment (chauhang) [inline examples/asr_rnnt_emformer/01_create_model_archive.sh:3]: Please update to use the current latest version of TorchServe

File: examples/asr_rnnt_emformer/handler.py (hunk 3 above)
Comment (chauhang) [reply examples/asr_rnnt_emformer/handler.py:55]: For audio models, batching may not make sense in all cases. Will be good to mention as a note in the readme that this example does not support batching and verify the config properties match

**Review by chauhang — CHANGES_REQUESTED** [review]
Thanks for the PR, can be merged with few minor changes for torchserve version and readme updates

File: examples/asr_rnnt_emformer/00_save_jit_model.sh (hunk 4 above)
Comment (msaroufim) [inline examples/asr_rnnt_emformer/00_save_jit_model.sh:3]:
```suggestion
CONTAINER=pytorch/torchserve:0.8.2-cpu
```

File: examples/asr_rnnt_emformer/01_create_model_archive.sh (hunk 5 above)
Comment (msaroufim) [inline examples/asr_rnnt_emformer/01_create_model_archive.sh:3]:
```suggestion
CONTAINER=pytorch/torchserve:0.8.2-cpu
```

**Review by agunapal — APPROVED** [review]

**Pull request merged**